### ISSUE TITLE
DrudeLangevinIntegrator

### ISSUE THREAD

**Issue by jchodera** [pandegroup/openmm#915]
@peastman : I'm curious about the origin of the [`DrudeLangevinIntegrator`](https://github.com/pandegroup/openmm/blob/master/plugins/drude/openmmapi/include/openmm/DrudeLangevinIntegrator.h#L41-L50).  

Is this something you came up with on your own, or is there a reference that you drew on to construct this integrator?  Is it as straightforward as replacing the two-temperature [Nose-Hoover integrator from Lamoureux and Roux](http://dx.doi.org/10.1063/1.1589749) with a Langevin integrator with two temperatures, or is there something more sophisticated going on here?  Did you need to use multiple timesteps as well?

**Comment by peastman** [comment]
No, this is the standard method used for simulations with Drude particles.  If I recall correctly, I roughly modeled it after the version in CHARMM.  Do a web search for "drude particle extended lagrangian" and you'll find tons of references to it.

**Comment by jchodera** [comment]
Huh.  The latest CHARMM documentation is here:
* [Drude oscillators](http://www.charmm.org/documentation/c39b1/drude.html)
* [Dynamics](http://www.charmm.org/documentation/c39b1/dynamc.html)

The dynamics documentation mentions a `VV2` integrator that provides velocity Verlet that works with Drude:
```
The VV2 is a velocity-Verlet integrator based on the
      operator-splitting technique.  It works in conjunction with the
      TPCONTROL command.  It allows temperature and pressure control
      [G. J. Martyna, M. E. Tuckerman, D. J. Tobias and M. L. Klein,
      Mol. Phys. 87, 1117 (1996)], and provides an efficient
      integration algorithm for polarizable force fields based on
      Drude oscillators [G. Lamoureux and B. Roux, J. Chem. Phys. 119,
      ???  (2003)].  It uses a separate "SHAKE" algorithm, called
      RATTLE/Roll.  See *note tpcontrol:(chmdoc/tpcontrol.doc)Dynamics.
```
but references that [Lamoureux and Roux](http://dx.doi.org/10.1063/1.1589749) paper that uses a Nose-Hoover method.

The [NAMD documentation](http://www.ks.uiuc.edu/Research/Drude/) seems to mention a Langevin integrator approach:
> The integration of the extended Lagrangian is performed using a dual thermostat to keep the Drude oscillators at a cold temperature T⋆ while the remaining warm degrees of freedom are maintained at a desired temperature T. In a previous implementation of the Drude model, this was achieved by designing a dual Nosé-Hoover thermostat acting on, and within, each nucleus-Drude pair [3].

but this also refers to the same paper:
> G. Lamoureux and B. Roux. Modeling induced polarization with classical Drude oscillators: Theory and molecular dynamics simulation algorithm. Journal Of Chemical Physics, 119(6):3025–3039, 2003.

**Comment by jchodera** [comment]
Maybe we could tag this as `documentation`?  I think it's just a matter of documenting the provenance of this particular integrator in the user guide.

**Comment by peastman** [comment]
Github appears to have a hardcoded list of six possible labels: bug, duplicate, enhancement, invalid, question, and wontfix.  It won't let me add a different label that isn't in the list.  Is there some way to create new labels?

**Comment by jchodera** [comment]
Yes!

Go to "Issues", and then click on the "Labels" tab.  From there, you can edit, delete, or create new labels.

**Comment by jchodera** [comment]
Similarly, you can use the "Milestones" tab to delineate new milestone releases, such as "OpenMM 6.3", "OpenMM 6.4", "OpenMM 7.0", or "intedeterminate future", etc.

**Comment by peastman** [comment]
Thanks!  That was an extremely well hidden user interface.  Even with your instructions it took me a while to find it.

**Comment by jchodera** [comment]
I don't disagree with your assessment.  :)
